src/test/create-user.test.ts

```diff
@@ -0,0 +1,89 @@
+import { CryptoService } from '@core/crypto';
+import { CRYPTO_KEY_LENGTH, CRYPTO_SECRET } from '@core/env-conf';
+import { UserDataSource } from '@db/source';
+import { CreateUserUseCase } from '@domain/mutation';
+import { expect } from 'chai';
+import Container from 'typedi';
+
+describe('GraphQL -> Create User - use-case', () => {
+  let createUser: CreateUserUseCase;
+  let repository: UserDataSource;
+  let cryptoService: CryptoService;
+
+  beforeEach(() => {
+    const secret = Container.get(CRYPTO_SECRET);
+    const key_length: number = Container.get(CRYPTO_KEY_LENGTH);
+
+    repository = new UserDataSource();
+    cryptoService = new CryptoService(secret, key_length);
+    createUser = new CreateUserUseCase(repository, cryptoService);
+  });
```

Comment:
como comentado em daily:
```suggestion
  before(() => {
    createUser = Container.get(CreateUserUseCase);
  });
```